corehq/apps/export/views.py

```diff
@@ -1550,7 +1550,10 @@ def post(self, request, *args, **kwargs):
             return self.get(request, *args, **kwargs)
 
         aggregate = DataFile.objects.filter(domain=self.domain).aggregate(total_size=Sum('content_length'))
-        if aggregate['total_size'] + request.FILES['file'].size > MAX_DATA_FILE_SIZE_TOTAL:
+        if (
+            aggregate['total_size'] and
+            aggregate['total_size'] + request.FILES['file'].size > MAX_DATA_FILE_SIZE_TOTAL
+        ):
```

Comment:
if i understand this correctly, i think the way this is right now it would always let you upload any size you want the first time, since this statement would always evaluate to False?